tests/e2e/visual/messages/messages-overview.wdio-spec.js

```diff
@@ -0,0 +1,131 @@
+const commonPage = require('@page-objects/default/common/common.wdio.page');
+const loginPage = require('@page-objects/default/login/login.wdio.page');
+const messagesPage = require('@page-objects/default/sms/messages.wdio.page');
+const dataFactory = require('@factories/cht/generate');
+const utils = require('@utils');
+const { generateScreenshot, resizeWindowForScreenshots, isMobile } = require('@utils/screenshots');
+const modalPage = require('@page-objects/default/common/modal.wdio.page');
+
+describe('Messages Overview', () => {
+  const docs = dataFactory.createHierarchy({
+    name: 'Visual Messaging User',
```

Comment:
```suggestion
    name: 'Kiambu',
```

Can you please change this to be more real?